src/js/tagFilter.js

```diff
@@ -1,12 +1,8 @@
-import loadData from "./loadData.js"
-
-const tagParent = document.querySelector("#btn-container");
-
 /*
 Loops through all challenges and saves all labels/tags in an array
 */
 async function getTags(loadData) {
-  let data = await loadData();
+  let data = await loadData;
```

Comment:
This is confusing to me, because I interpret `loadData` as being a function. The `await` keyword is used for promises, often promises returned directly by functions. But if `loadData` is a promise, it's not a function.

To me `await loadData()` looks more sensible than `await loadData`. Of course, if `loadData` is a promise and not a function, your way is still correct, but then it should probably be renamed to reflect that it's not a function.

To me, all of this just indicates that the communication between different parts of the code is a bit incoherent and messy. I've tried to understand it, but it's taking too much effort, which is a bad sign.